src/Microsoft.DocAsCode.MarkdownLite/Regexes.cs

```diff
@@ -48,7 +48,7 @@ public static class Inline
             public static readonly Regex Escape = new Regex(@"^\\([\\`*{}\[\]()#+\-.!_>])", RegexOptionCompiled, RegexTimeOut);
             public static readonly Regex Comment = new Regex(@"^<!--(?:[^-]|-(?!->))*-->", RegexOptionCompiled, RegexTimeOut);
             public static readonly Regex AutoLink = new Regex(@"^<([^ >]+(@|:\/)[^ >]+)>", RegexOptionCompiled, RegexTimeOut);
-            public static readonly Regex CodeElement = new Regex(@"^\<code\>[\s\S]*?\</code\>", RegexOptionCompiled | RegexOptions.IgnoreCase, RegexTimeOut);
+            public static readonly Regex PreElement = new Regex(@"^\<pre\>[\s\S]*?\</pre\>", RegexOptionCompiled | RegexOptions.IgnoreCase, RegexTimeOut);
```

Comment:
In GitHub, it gets bold in New File Preview, but doesn't in comment Preview here... It also gets bold in commonmark: http://spec.commonmark.org/dingus/?text=%3Ccode%3E*p%3D*q%3C%2Fcode%3E